docs/framework/install/on-windows-10.md

```diff
@@ -1,49 +1,50 @@
 ---
 title: Install the .NET Framework on Windows 10
-description: Learn how to install the .NET Framework on Windows 10.
+description: Learn how to install the .NET Framework on Windows 10 or Windows Server 2016.
 author: rlander
 ms.author: mairaw
 keywords: .NET Framework, Install
-ms.date: 10/16/2017
+ms.date: 11/17/2017
 ms.topic: article
 ms.custom: "updateeachrelease"
 ms.prod: .net-framework
 ---
 # Install the .NET Framework on Windows 10 and Windows Server 2016
 
-The .NET Framework is required to run many applications on Windows. You can use the following instructions to install it. You may have arrived on this page after trying to run an application and seeing the following dialog on your machine.
+The .NET Framework is required to run many applications on Windows. The instructions on this article should help you install the .NET Framework versions that you need. The [.NET Framework 4.7.1](https://www.microsoft.com/download/details.aspx?id=56115&desc=dotnet47) is the latest available version.
 
-![This application could not be started](./media/this-application-could-not-be-started.png)
+You may have arrived on this page after trying to run an application and seeing a dialog on your machine similar to the following image:
 
-These instructions will help you install the .NET Framework versions you need. The [.NET Framework 4.7.1](https://www.microsoft.com/en-us/download/details.aspx?id=56115&desc=dotnet47) is the latest version.
+![This application could not be started](./media/this-application-could-not-be-started.png)
 
 ## .NET Framework 4.7.1
 
-* [Download the .NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral)
+> [!div class="button"]
+[Download .NET Framework 4.7.1](https://www.microsoft.com/net/download/thank-you/net471?utm_source=ms-docs&utm_medium=referral)
 
 The .NET Framework 4.7.1 is included with:
```

Comment:
I think that it might be useful to expand this to note that because it's pre-installed, you don't have to install it or an earlier version of the .NET Framework. Although we find it obvious, I don't think our customers do.